docs/build.md

```diff
@@ -0,0 +1,128 @@
+# Build instructions
+
+The pipeline requires Python version 3.5.
+
+## Python 3.5 installation (optional)
+The instructions below will be useful:
+* In case you have a newer version and need to run the pipeline you can follow the next steps to install Python 3.5
+without replacing you default `python`;
+* If you are running the pipeline on the `ebi-cli` cluster, which currently only supports Python 3.4.
+
+Adjust `VERSION` and `INST_PATH` if needed.
+
+```bash
+VERSION=3.5.6
+INST_PATH=${HOME}/software/python-${VERSION}
+mkdir -p ${INST_PATH}
+cd ${INST_PATH}
+wget https://www.python.org/ftp/python/${VERSION}/Python-${VERSION}.tgz
+tar zxfv Python-${VERSION}.tgz
+mv Python-${VERSION}/* .
+rmdir Python-${VERSION}
+```
+
+### Root installation
+
```

Comment:
Our Markdown style is usually no blank lines between a heading and the following paragraph, and one blank line between the end of a section and the next heading. Applies to other sections in other files too.
```suggestion
```

Reply:
Fixed all four Markdown files to adhere to this style